readme.md

```diff
@@ -82,7 +94,12 @@ var Panda = function(name, age) {
 
 Your Answer:
 ```text
-
+Panda.prototype={
+  eat_bamboo: function(){
+    var self = this;
+    self.num_bamboo_eaten += 1;
```

Comment:
Totally works, self is not totally necessary here since we don't change context but not hurting anything